docs/pages/inboxes/send-messages.md

```diff
@@ -420,6 +484,28 @@ For example:
 
 :::code-group
 
+```tsx [Browser]
+// Update disappearing message settings
+await conversation.updateDisappearingMessageSettings({
+  disappearStartingAtNs: 1738620126404999936,
+  retentionDurationInNs: 1800000000000000
+})
```

Comment:
`updateDisappearingMessageSettings` => `updateMessageDisappearingSettings`

the method accepts 2 arguments, so to convert your example would be:
```
await conversation.updateMessageDisappearingSettings(1738620126404999936n, 1800000000000000n)
```